.github/workflows/shortlog.yml

```diff
@@ -0,0 +1,23 @@
+name: check contributor list
```

Comment:
Do we still need this? Perhaps it would be better to leave it to the release workflow so we have less stuff to maintain?

Reply:
Better the catch it early than late.

We want contributors to add themselves to the list (and mailmap as necessary) as this infers agreement to the CLA (and the mailmap associates the commit with the contributor list). By release time it's a little too late but better to find out then than never.